sensor_msgs/Image cframe_image
float64[] cframe_values

